### DISCUSSION TITLE
Error in the log "Unsupported Operation"

### DISCUSSION THREAD

**Discussion by xbukorg** [ACINQ/eclair#2542] [Q&A] [answered]
Hi,

What does it mean error in the Eclair log

Exception in thread "OkHttp Dispatcher" java.lang.UnsupportedOperationException: Couldn't connect to the proxy server, uri: http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/, socket: 127.0.0.1:9050

and how can I fix it?

Thanx&Regards

**Comment by rorp** [comment]
That means that eclair cannot connect to `mempool.space` over Tor to get the latest blockchain info.

There are 3 ways to "fix" this:

1. start Tor (recommended)
2. remove `mempool.space` from the watchdog source list (less recommended):
 ```
eclair.blockchain-watchdog.sources = [
      "bitcoinheaders.net",
      "blockcypher.com",
      "blockstream.info"
]
```
3. set `eclair.socks5.use-for-watchdogs = false` to disable Tor for the blockchain watchdogs (not recommended)

**Reply by xbukorg** [reply]
Thanks for the answerer but Tor is running and error is logged in this case. 

My Eclair config is:
eclair.tor.enabled=true
eclair.tor.protocol-version="v3"
eclair.tor.password=XXXXXXXXXXXXX
eclair.socks5.enabled=true
eclair.socks5.port=YYYY
eclair.socks5.randomize-credentials=true

For other Tor node work correctly.

**Reply by t-bast** [reply]
As explained in the comment below, Tor is not reliable, that's why you see this kind of issues.
In most cases you simply cannot "fix them": if you're unable to connect to a remote site over Tor, that's outside of your control.

**Comment by rorp** [comment, marked as answer]
Also, a thing to consider. Tor is not the most reliable network, so if you see this kind of errors just occasionally, 
you don't really need to fix them. Most likely that's just intermittent Tor connectivity issues, that eventually resolve themselves.
